include/realizations/catchment/Tshirt_C_Realization.hpp

```diff
@@ -8,7 +8,17 @@
 #include "GiuhJsonReader.h"
 #include "tshirt_c.h"
 #include <memory>
+#include <set>
```

Comment:
I didn't see anything in this change set that used `set`, if it is not used, should be be removed.

Reply:
Yeah, I think I started to do something in the changes that used it, but then later went with a different approach.  Looks like I just forgot to take that out.  Fixed in `066bad4f3701`.